index.css

```diff
@@ -58,6 +62,82 @@ body {
   padding: 0.75rem;
 }
 
+.add-product-form {
+  display: flex;
+  margin: 1.5rem 2rem;
+}
+
+.add-product-form__label {
+  margin-bottom: .5rem;
+}
+
+.add-product-form__dropdown {
+  padding: .7rem 5rem .7rem .7rem;
+  background: var(--grey);
+  border-color: var(--grey);
+  color: var(--light-black);
+}
+
+
+.add-product-form__dropdown-item {
+  font-size: 16px;
+  padding: .1rem;
+}
+
+.add-product-form__wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-right: 3rem;
+}
+
+.add-product-form__wrapper--result {
+  margin-top: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+}
+
+.add-product-form__input {
+
+  border: solid var(--grey);
+  padding: .7rem;
+  background: var(--grey);
+  color: var(--light-black);
+}
+
+.add-product-form__result {
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.add-product-form__result--kcal-outcome {
+  font-weight: bold;
+  margin-bottom: .5rem;
+}
+
+.add-product-form__result--kcal-info {
+  color: var(--shadow-grey);
```

Comment:
Wiem, że tak jest na Figmie, ale to mój błąd. Warto tutaj użyć `--dark-grey` i usunąć `--shadow-grey` w ogóle, żeby zachować konwencję.